docs/misc/deprecated.md

```diff
@@ -15,6 +15,10 @@ The following list of features are deprecated.
 ### Old Command Line Options
 **Deprecated In Release: [v1.8.0](/release-notes/#docker-engine-1-8-0)**
 
+The flags `-d` and `--daemon` are deprecated in favor of the `daemon` subcommand:
+
+    docker daemon -H ...
+
```

Comment:
if they install the packages the init scripts are updated, afaik. I think the explanation shouldn't go here, this is just a list of things we're deprecating, isn't it?